app/controllers/course/experience_points_disbursement_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+class Course::ExperiencePointsDisbursementController < Course::ComponentController
+  include Course::UsersBreadcrumbConcern
+  load_resource :experience_points_disbursement, class: Course::ExperiencePointsDisbursement.name
+
+  def new # :nodoc:
+    @experience_points_disbursement.course = current_course
```

Comment:
change this to be a method on the disbursement model, because this has side-effects